### DISCUSSION TITLE
How to make the mod actually function as a client sided mod?

### DISCUSSION THREAD

**Discussion by kill-joy-gaming** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#3] [Q&A] [answered]
i'm only using this mod for the range increase, as stated this is supposed to be fully client sided but when I try to join a lobby in lethal company I'm unable to until I remove the mod. does anyone know a way to fix this?

**Comment by MrHydralisk** [comment]
And ig you turned off everything except MapRangeRBFlashEnabled in Config, right? Also is it happening on v45 or new version of the Lethal Company?

**Reply by kill-joy-gaming** [reply]
yes, i turned everything off apart from the zoom on the radar, and its on the newest version of the game

**Reply by MrHydralisk** [reply]
I heard from other devs that LC added some new way to check if modded lobby or not with new version of the game, so it possible that it due to it. So it prob take some time to adjust it for new ver.

**Reply by kill-joy-gaming** [reply]
alright
guess i wont get my upgrade yet D:

**Reply by MrHydralisk** [reply]
Btw, was there any error logged in Player.Log file, when you tried to join with this mod?

**Reply by MrHydralisk** [reply]
Nvm, I think I figured how to fix it.

**Reply by MrHydralisk** [reply, marked as answer]
Should be fixed now in v1.5.3.
